e2e-tests/portal/cypress/support/portal-api/api.js

```diff
@@ -45,7 +45,7 @@ module.exports.listAllDeals = (token) => {
     body: JSON.stringify(body),
   }).then((resp) => {
     expect(resp.status).to.equal(200);
-    return resp.body.data.deals.deals;
+    return resp.body.data.allDeals.deals;
```

Comment:
for the future, is this not counter productive? 'allDeals.deals'

Reply:
I agree it is a bit counter intuitive - it's handling the response from graphQL.

`response.body.data`  = response object, but from here we get this because of the current schema.

```js
response.body.data = {
  allDeals: {
    count: 100,
    deals: [ ... ],
  },
};
```

`allDeals` is the query name in graphql schema